try {
    Integer a = 5 / 0;
} catch(Exception e) {
    System.debug(e.getMessage());
}
